Codeinwp/neve
# feat: update TPC banner image

### Summary

Updated the image for the TPC banner.

### Will affect the visual aspect of the product

YES

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/234254977-39b6dc81-843f-4d8d-aa19-32f11ce07fb3.png)


### Test instructions

1. Create a new instance with Neve
2. Check inside the dashboard that the banner from the screenshot ☝🏼 has the updated image.

## Check before Pull Request is ready:



Closes #3919.
